src/Constants.m

```diff
@@ -518,10 +518,10 @@
         InitT6 = 0;
         Tss = InitT0;
     end
-    if nanmean(Ta_msr) < 0
+    if mean(Ta_msr, "omitnan") < 0
```

Comment:
This changes were needed because `nanmean` is not recommended in Matlab, and deprecated in Octave.